projects/noloan/common/src/main/java/hasadna/noloan/common/SmsMessages.java

```diff
@@ -42,7 +42,7 @@ public void suggestMessage(SmsMessage smsMessage) {
     // 1. Add user as a "suggester"
     int index = searchDbMessage(smsMessage);
     if ((index != -1)
-        && dbMessages
+        && !dbMessages
```

Comment:
Why this change?

Reply:
It's a bug fix. I think you accidentally removed the `!` here: 
https://github.com/hasadna/monorepo/commit/a14fceba8dc579302d1342bcfb0e83195a269b21#diff-e2258fc75a728aa3ae8880a986424ae0L61

It prevents a user from suggesting a spam more than once.

Reply:
Opss?